src/Variant/DefaultVariantHandler.php

```diff
@@ -11,6 +11,8 @@
 
 final class DefaultVariantHandler implements VariantHandler
 {
+    const VARIANT_HASH_SEED = 86028157;
```

Comment:
Where is this value from? Also, please add visibility (`private`, `protected`, `public`)

Reply:
Marked it as private

This is just an arbitrary but large number used for the seed. We have the same bug in all of our SDKs so we're standardising on this

I'd love a way to communicate this out with the client spec but that isn't really possible with the client specs design right now